views/genres.pug

```diff
@@ -1,14 +1,14 @@
 extends layout
 
 append head
-    //- add page specific styles by appending to the head
-    link(rel="stylesheet" href="/stylesheets/allGenres.css")
-    //- add page specific js
+    link(rel="stylesheet" href="/stylesheets/allGensPubs.css")
     script(src="/javascripts/index.js" type="module" defer)
 block content
-    .genres-container
-        h1 All Genres
-        .genres-container__thumbnails
-            each genre in genres
-                div
-                    a(href=`/genres/${genre.id}`) #{genre.genreName}
+    body
```

Comment:
I don't think this body tag is needed, the body tag already exists in the layout.pug that is extended.

Reply:
the body tag allowed me to style the whole page in line with the rest of the aesthetic